examples/Creating_slides_with_Assistants_API_and_DALL-E3.ipynb

```diff
@@ -0,0 +1,786 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Making slides with the Assistants API and DALL-E3\n",
```

Comment:
fix DALLE-3 spelling/spacing

Reply:
Changed to DALL·E-3 per other notebooks (e.g.: https://cookbook.openai.com/articles/what_is_new_with_dalle_3)